src/Navbar.js

```diff
@@ -93,13 +124,19 @@ class Navbar extends React.Component {
   }
 
   getChildContext() {
-    const { bsClass, expanded } = this.props;
+    const { bsClass, expanded, onSelect, toggleOnSelect } = this.props;
+    const mobileToggle = () => {
+      if (typeof(window) !== 'undefined' && window.innerWidth < 768) {
```

Comment:
Yeah it's brittle hard-coded as is.

One approach I can think of dealing with is is exposing a prop `toggleOnSelectMaxWidth`. It's of `React.PropTypes.number` and defaults to 768. So then when `toggleOnSelect == true` the window.innerWidth check would compare with `toggleOnSelectMaxWidth`.

Or `toggleOnSelect` could be the numeric value itself, and default to 0, so it's falsey when not set.